movie_rec/sql_scripts/upload_movies.py

```diff
@@ -0,0 +1,51 @@
+import sys
+import json
+from optparse import OptionParser
+import psycopg2
+
+parser = OptionParser()
+parser.add_option("-i", "--input", dest="input", metavar="FILE",
+                  help="name of upload file")
+(options, args) = parser.parse_args()
+
+conn = psycopg2.connect("dbname=movie_rec")
+cur = conn.cursor()
+description = None
+year = None
+title = None
+image_url = None
+
+# for line in sys.stdin:
+with open(options.input) as f:
+    for line in f:
+
+        data = json.loads(line.rstrip())
+        url_handle = data['movie_url_handle']
+
+        if 'description' in data:
+            description = data['description']
```

Comment:
If we simplify this (and below) to `description = data.get('description')`, it defaults to `None` if key doesn't exist right? Then don't need to initiate values to `None` in lines 13-16, and don't need to reset in lines 43-46.